api/src/test/java/org/openmrs/api/OrderSetServiceTest.java

```diff
@@ -348,9 +348,8 @@ private OrderSet orderSetBuilder(boolean orderSetRetired, boolean orderSetMember
 		orderSetMember.setDateCreated(new Date());
 		orderSetMember.setRetired(orderSetMemberRetired);
 		orderSetMember.setOrderSet(orderSet);
-		
-		List<OrderSetMember> orderSetMembers = new ArrayList<>();
-		orderSetMembers.addAll(Collections.singletonList(orderSetMember));
+
+		List<OrderSetMember> orderSetMembers = new ArrayList<>(Arrays.asList(orderSetMember));
```

Comment:
Did you intentionally change this from Collections.singletonList?

Reply:
By accident while resolving conflicts. Fixed already, thanks :)